src/promptflow/tests/test_configs/datas/load_data_cases/colors.csv

```diff
@@ -0,0 +1,4 @@
+name,id_text,id_int,id_float
+Red,"1.0",1,1.0
+Blue,"3.0",3,3.0
+Yellow,"2.0",3,2.0
```

Comment:
You can add some Japanese/Arabic columns maybe?

Reply:
will do this in future PR, this PR targets to fix the wrong type infer bug